### PR TITLE
Removing MuTect1

### PR DESCRIPTION
**Pull request by MaxUlysse**

### CODE REVIEW

File: germlineVC.nf [hunk 1]
```diff
@@ -402,12 +384,11 @@ process ConcatVCF {
     set variantCaller, idPatient, idSampleNormal, idSampleTumor, file("*.vcf.gz") into vcfConcatenated
     file("*.vcf.gz.tbi") into vcfConcatenatedTbi
 
-  when: ( 'haplotypecaller' in tools || 'mutect1' in tools || 'mutect2' in tools || 'freebayes' in tools ) && !params.onlyQC
+  when: 'haplotypecaller' in tools && !params.onlyQC
```
Comment (szilvajuhos) [inline germlineVC.nf]: I am a bit confused here, so it is germline calling, so all the somatic callers are removed, if I am correct.

File: germlineVC.nf [hunk 2]
```diff
@@ -651,7 +632,6 @@ def defineDirectoryMap() {
     'haplotypecaller'  : "${params.outDir}/VariantCalling/HaplotypeCaller",
     'gvcf-hc'          : "${params.outDir}/VariantCalling/HaplotypeCallerGVCF",
     'manta'            : "${params.outDir}/VariantCalling/Manta",
-    'mutect1'          : "${params.outDir}/VariantCalling/MuTect1",
     'mutect2'          : "${params.outDir}/VariantCalling/MuTect2",
```
Comment (szilvajuhos) [inline germlineVC.nf]: So Mutect2 also can be removed.

File: germlineVC.nf [hunk 3]
```diff
@@ -679,7 +659,6 @@ def defineToolList() {
     'freebayes',
```
Comment (szilvajuhos) [inline germlineVC.nf]: ditto freebayes though that can be used for germline, so, if somebody wants to have that, s/he can still use it.

Comment (MaxUlysse) [reply germlineVC.nf]: Can we use Freebayes for germline?
We're not doing that yet.
Which settings should we use?

**Pull request closed without merging**